.github/workflows/monorepo.yaml

```diff
@@ -29,4 +29,27 @@ jobs:
         run: |
           cd monorepo/google-cloud-java
           git remote add monorepo https://${{ github.actor }}:${{ github.token }}@github.com/${{ github.repository }}.git
-          git push -f monorepo main:monorepo_script_output
\ No newline at end of file
+          git push -f monorepo main:monorepo_script_output
+
+  build-3repos:
+    runs-on: ubuntu-latest
+    name: Running bootstrap.sh with only 3 selected repositories
+    steps:
+    - uses: actions/checkout@v3
+    - name: Truncate repos.txt for 3 repositories
+      run: |
+        cat << EOL > repos.txt
+        java-accessapproval
+        java-dataform
```

Comment:
In your pull request, replace java-dataform with the module that somehow continues to get SNAPSHOT version.